docsk8s/index.asciidoc

```diff
@@ -42,9 +42,15 @@ include::setting-up/ls-k8s-secure.asciidoc[]
 // Administering
 include::administering/ls-k8s-administering.asciidoc[]
 
-// Set up stack monitoring
+// Stack Monitoring
 include::administering/ls-k8s-stack-monitoring.asciidoc[]
 
+// Stack Monitoring external
+include::administering/ls-k8s-stack-monitoring-external.asciidoc[]
+
+// Stack Monitoring Elastic Cloud
+include::administering/ls-k8s-stack-monitoring-cloud.asciidoc[]
+
 // Upgrade Logstash
```

Comment:
"Upgrading" should probably move up a level.  But we can wait and see, and maybe make that change in another pass.